### ISSUE TITLE
Fonts not found

### ISSUE THREAD

**Issue by egli** [sbsdev/catalog#40]
When running as a jar the fonts in the file system are not found

Either
1. make sure file system fonts are found or
2. package the fonts in resources

**Comment by egli** [comment]
The problem is not with the fonts. We get the following exception:
```
SEVERE: Could not set base URL for svg
java.lang.IllegalArgumentException: URI is not absolute
	at java.net.URI.toURL(URI.java:1088)
	at org.apache.fop.fo.extensions.svg.SVGElement.getDimension(SVGElement.java:77)
	at org.apache.fop.fo.flow.InstreamForeignObject.prepareIntrinsicSize(InstreamForeignObject.java:112)
	at org.apache.fop.fo.flow.InstreamForeignObject.getIntrinsicWidth(InstreamForeignObject.java:125)
	at org.apache.fop.layoutmgr.inline.AbstractGraphicsLayoutManager.getInlineArea(AbstractGraphicsLayoutManager.java:61)
	at org.apache.fop.layoutmgr.inline.AbstractGraphicsLayoutManager.getNextKnuthElements(AbstractGraphicsLayoutManager.java:116)
	at org.apache.fop.layoutmgr.inline.LineLayoutManager.collectInlineKnuthElements(LineLayoutManager.java:698)
	at org.apache.fop.layoutmgr.inline.LineLayoutManager.getNextKnuthElements(LineLayoutManager.java:627)
	at org.apache.fop.layoutmgr.BlockLayoutManager.getNextChildElements(BlockLayoutManager.java:141)
	at org.apache.fop.layoutmgr.BlockStackingLayoutManager.getNextKnuthElements(BlockStackingLayoutManager.java:290)
	at org.apache.fop.layoutmgr.BlockLayoutManager.getNextKnuthElements(BlockLayoutManager.java:113)
	at org.apache.fop.layoutmgr.BlockLayoutManager.getNextKnuthElements(BlockLayoutManager.java:105)
	at org.apache.fop.layoutmgr.BlockContainerLayoutManager$BlockContainerBreaker.getNextKnuthElements(BlockContainerLayoutManager.java:671)
	at org.apache.fop.layoutmgr.AbstractBreaker.getNextBlockList(AbstractBreaker.java:770)
	at org.apache.fop.layoutmgr.AbstractBreaker.getNextBlockList(AbstractBreaker.java:743)
	at org.apache.fop.layoutmgr.AbstractBreaker.doLayout(AbstractBreaker.java:389)
	at org.apache.fop.layoutmgr.BlockContainerLayoutManager.getNextKnuthElementsAbsolute(BlockContainerLayoutManager.java:535)
	at org.apache.fop.layoutmgr.BlockContainerLayoutManager.getNextKnuthElements(BlockContainerLayoutManager.java:212)
	at org.apache.fop.layoutmgr.BlockContainerLayoutManager.getNextKnuthElements(BlockContainerLayoutManager.java:186)
	at org.apache.fop.layoutmgr.LocalBreaker.getNextKnuthElements(LocalBreaker.java:107)
	at org.apache.fop.layoutmgr.AbstractBreaker.getNextBlockList(AbstractBreaker.java:770)
	at org.apache.fop.layoutmgr.AbstractBreaker.getNextBlockList(AbstractBreaker.java:743)
	at org.apache.fop.layoutmgr.AbstractBreaker.doLayout(AbstractBreaker.java:389)
	at org.apache.fop.layoutmgr.StaticContentLayoutManager.doLayout(StaticContentLayoutManager.java:140)
	at org.apache.fop.layoutmgr.PageSequenceLayoutManager.layoutSideRegion(PageSequenceLayoutManager.java:232)
	at org.apache.fop.layoutmgr.PageSequenceLayoutManager.finishPage(PageSequenceLayoutManager.java:240)
	at org.apache.fop.layoutmgr.PageSequenceLayoutManager.activateLayout(PageSequenceLayoutManager.java:147)
	at org.apache.fop.area.AreaTreeHandler.endPageSequence(AreaTreeHandler.java:267)
	at org.apache.fop.fo.DelegatingFOEventHandler.endPageSequence(DelegatingFOEventHandler.java:114)
	at org.apache.fop.accessibility.fo.FO2StructureTreeConverter.endPageSequence(FO2StructureTreeConverter.java:148)
	at org.apache.fop.fo.pagination.PageSequence.endOfNode(PageSequence.java:139)
	at org.apache.fop.fo.FOTreeBuilder$MainFOHandler.endElement(FOTreeBuilder.java:362)
	at org.apache.fop.fo.FOTreeBuilder.endElement(FOTreeBuilder.java:190)
	at com.sun.org.apache.xml.internal.serializer.ToXMLSAXHandler.endElement(ToXMLSAXHandler.java:265)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:609)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1782)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2967)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:602)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:505)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:842)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:771)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:643)
	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transformIdentity(TransformerImpl.java:632)
	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:728)
	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:343)
	at catalog.layout.fop$generate_pdf_BANG_.invokeStatic(fop.clj:1093)
	at catalog.layout.fop$generate_pdf_BANG_.invoke(fop.clj:1060)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at iapetos.collector.fn$wrap_instrumentation$fn__4403.doInvoke(fn.clj:42)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at iapetos.collector.fn$wrap_instrumentation$fn__4406.doInvoke(fn.clj:44)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at iapetos.collector.fn$wrap_instrumentation$fn__4409.doInvoke(fn.clj:46)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at iapetos.collector.fn$wrap_instrumentation$fn__4411.doInvoke(fn.clj:48)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at iapetos.collector.fn$wrap_instrumentation$fn__4413.doInvoke(fn.clj:50)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at catalog.web.views$neu_im_sortiment.invokeStatic(views.clj:150)
	at catalog.web.views$neu_im_sortiment.invoke(views.clj:142)
	at catalog.routes.home$home_routes$fn__21634.invoke(home.clj:36)
	at reitit.ring.coercion$fn__21584$fn__21586$fn__21587.invoke(coercion.cljc:40)
	at muuntaja.middleware$wrap_params$fn__5610.invoke(middleware.clj:52)
	at muuntaja.middleware$wrap_format$fn__5614.invoke(middleware.clj:73)
	at catalog.middleware$wrap_formats$fn__6500.invoke(middleware.clj:38)
	at ring.middleware.anti_forgery$wrap_anti_forgery$fn__3664.invoke(anti_forgery.clj:94)
	at reitit.ring$ring_handler$fn__22121.invoke(ring.cljc:329)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.lang.Var.invoke(Var.java:384)
	at ring.middleware.flash$wrap_flash$fn__6106.invoke(flash.clj:39)
	at ring.adapter.undertow.middleware.session$wrap_undertow_session$fn__6057.invoke(session.clj:77)
	at ring.middleware.keyword_params$wrap_keyword_params$fn__6144.invoke(keyword_params.clj:53)
	at ring.middleware.nested_params$wrap_nested_params$fn__6194.invoke(nested_params.clj:89)
	at ring.middleware.multipart_params$wrap_multipart_params$fn__6300.invoke(multipart_params.clj:171)
	at ring.middleware.params$wrap_params$fn__6316.invoke(params.clj:67)
	at ring.middleware.cookies$wrap_cookies$fn__5874.invoke(cookies.clj:214)
	at ring.middleware.absolute_redirects$wrap_absolute_redirects$fn__6440.invoke(absolute_redirects.clj:47)
	at ring.middleware.resource$wrap_resource_prefer_resources$fn__6336.invoke(resource.clj:25)
	at ring.middleware.content_type$wrap_content_type$fn__6404.invoke(content_type.clj:34)
	at ring.middleware.default_charset$wrap_default_charset$fn__6420.invoke(default_charset.clj:31)
	at ring.middleware.not_modified$wrap_not_modified$fn__6386.invoke(not_modified.clj:61)
	at ring.middleware.x_headers$wrap_x_header$fn__6077.invoke(x_headers.clj:22)
	at ring.middleware.x_headers$wrap_x_header$fn__6077.invoke(x_headers.clj:22)
	at ring.middleware.x_headers$wrap_x_header$fn__6077.invoke(x_headers.clj:22)
	at iapetos.collector.ring$run_instrumented$fn__4546.invoke(ring.clj:135)
	at iapetos.collector.ring$safe.invokeStatic(ring.clj:129)
	at iapetos.collector.ring$safe.invoke(ring.clj:126)
	at iapetos.collector.ring$run_instrumented.invokeStatic(ring.clj:135)
	at iapetos.collector.ring$run_instrumented.invoke(ring.clj:131)
	at iapetos.collector.ring$wrap_instrumentation$fn__4556.invoke(ring.clj:173)
	at iapetos.collector.ring$wrap_metrics_expose$fn__4565.invoke(ring.clj:194)
	at catalog.middleware$wrap_internal_error$fn__6494.invoke(middleware.clj:17)
	at ring.adapter.undertow$undertow_handler$fn$reify__23551.handleRequest(undertow.clj:40)
	at io.undertow.server.session.SessionAttachmentHandler.handleRequest(SessionAttachmentHandler.java:68)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:387)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:841)
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2019)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1558)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1449)
	at java.lang.Thread.run(Thread.java:748)

Alternate text is missing on instream-foreign-object. (See position 1:20400)
```
There is a [SO answer](https://stackoverflow.com/a/49451739) that seems to be related. They say that omitting or changing the default base URI for resolving resource URIs might help.

**Comment by egli** [comment]
Fixed by 36264fbf7a7736a089b472cc2a98537db6f00888
